lm75bd/lm75bd.c

```diff
@@ -27,7 +31,26 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
+  uint8_t tempReg[1]= {0};
+  i2cSendTo(LM75BD_OBC_I2C_ADDR, tempReg, 1);
+  uint8_t tempData[2] = {0,0};
```

Comment:
nit: The short form for initializing an array to 0s is the following. This is particularly useful if you have a larger array.

```suggestion
  uint8_t tempData[2] = {0};
```

In C, if you partially initialize an array (i.e. you specify the value for some of the elements in the array), then the rest of the elements will be given a value of 0.